conda/recipes/rapids_cuda_patched_dependencies/meta.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 39) %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.5').split('.')[:2]) %}
+{% set cuda_major=cuda_version.split('.')[0] %}
+
+package:
+  name: rapids_cuda_patched_dependencies
```

Comment:
What do we think of this name?

@bdice pruposed `rapids_cuda_includes` via slack which I like more

Reply:
`rapids` -- It's our package.
`cuda` -- It's CUDA-related libraries but not CUDA itself. We could theoretically have includes that are not CUDA, too. Maybe we drop this word.
`includes` -- We should emphasize that this is a header-only package.

So perhaps `rapids_includes` or `rapids_core_dependencies` or `rapids_common_includes` or a variation on those.

Reply:
I like `rapids_core_dependencies`, I will go with that for now